.github/workflows/push.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Test
       run: npm test
     - name: Publish
-      if: github.ref == 'refs/heads/master'
+      if: github.ref == 'refs/heads/master' && github.repository == 'multiformats/js-multiformats'
```

Comment:
huh? is this copypasta? intentionally left in?

Reply:
yup 🥵